index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/utility.css">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Micro-Portfolio</title>
+</head>
+<body>
+    <header class="container bg-gray">
+        <div class="flex header">
+            <span class="text-light">John Doe</span>
```

Comment:
- [ ] Kindly ensure your logo is a link using an `<a>` tag, this is a best practice as logos are used as links to the index page of the website.